examples/src/Syntax/DecomposeFormula/Config.hs

```diff
@@ -37,7 +46,14 @@ medium = DecomposeFormulaConfig
     , maxDepth = 6
     , availableAtoms = "ABCDE"
     , minAmountOfUniqueAtoms = 5
-    , allowArrowOperators = True
+    , operatorFrequencies = OperatorFrequencies
+      { and = 1
+      , or = 1
+      , impl = 0
+      , backImpl = 0
```

Comment:
Setting these two to `0` seems overzealous to me. Previously, `allowArrowOperators` was set to `True` here, which really allows all arrows to occur. The exception https://github.com/fmidue/logic-tasks/blob/8a9537b528e0981a048688b0df4f68cff07b3356/src/Tasks/DecomposeFormula/Quiz.hs#L23 applies only to the root of the syntax tree, not to all inner nodes.